models/backbones/efficientnet.py

```diff
@@ -0,0 +1,64 @@
+import torch
+import torchvision
+import torchvision.models as models
+from torchvision.models.efficientnet import FusedMBConv, MBConv
+
+from .abstract_backbone import AbstractBackbone
+
+
+def layer_props_en(layer):
+    if isinstance(layer, torchvision.ops.Conv2dNormActivation):
+        return layer[0].weight.shape[1], layer[0].stride[0]
+    if isinstance(layer, torch.nn.Conv2d):
+        return layer.weight.shape[1], layer.stride[0]
+    is_sequential = isinstance(layer, torch.nn.Sequential)
+    if is_sequential or isinstance(layer, MBConv) or isinstance(layer, FusedMBConv):
+        blocks = layer if is_sequential else layer.block
+        input_channels, stride = layer_props_en(blocks[0])
+        for layer in blocks[1:]:
+            _, layer_stride = layer_props_en(layer)
+            stride = max(stride, layer_stride)
+        return input_channels, stride
+    else:
+        return 0, 0
+
+
+def get_stride_features_and_filters_en(model):
```

Comment:
please, add docstring like:
def add(a, b):
    """
    Add two numbers and return the result.

    Args:
        a (int): The first number.
        b (int): The second number.

    Returns:
        int: The sum of the two numbers.
    """
    return a + b

or any other docstring format